packages/components/src/components/themed-scrollbars/themed-scrollbars.tsx

```diff
@@ -28,13 +28,13 @@ const ThemedScrollbars = ({
     is_only_horizontal_overlay,
     is_scrollbar_hidden,
     onScroll,
-    refSetter,
+    refSetter = null,
```

Comment:
Do we need to pass `null` to this?

Reply:
@niloo-fs I think yes, we have to pass into ref first value in order to not to pass undefined there, as this prop is optional

Reply:
Got it thank u @yauheni-kryzhyk-deriv